Week3/hanoi/src/App.java

```diff
@@ -1,31 +1,22 @@
-package Week3.hanoi.src;
-
 import java.util.Scanner;
 
 public class App {
   public static void main(String[] args) {
+    int numDisks = 0;  // The number of disks to move from fromBar to toBar.
 
-    // Get num of disks from System.in
-    // The number of disks should be more than 0.
-    int numDisks = 0;
+    // Get the number of disks from System.in
     Scanner sc = new Scanner(System.in);
```

Comment:
Handle a case about incorrect input such as hello and 12.2

Reply:
I put it inside the try so that the InputMismatchException log can be displayed.